### PR TITLE
Design tweaks : T247248

### PR DESCRIPTION
**Pull request by sharvaniharan**
**Phab:** https://phabricator.wikimedia.org/T247248

### CODE REVIEW

File: app/src/main/res/layout/group_find_references_in_page.xml [hunk 1]
```diff
@@ -10,8 +10,8 @@
     <TextView
         android:id="@+id/reference_label"
         android:layout_width="wrap_content"
-        android:layout_height="wrap_content"
-        android:layout_gravity="center_vertical"
+        android:layout_height="48dp"
+        android:gravity="center_vertical"
         android:layout_marginStart="16dp"
```
Comment (cooltey) [inline app/src/main/res/layout/group_find_references_in_page.xml]: Looks like the text does not align with the count text.

Would it be better if we do something like this:
```
        android:layout_height="wrap_content"
        android:layout_gravity="center_vertical"
        android:padding="16dp"
        android:background="?android:attr/selectableItemBackground"
```
Set padding, change height to `wrap_content`, change to the previous gravity setting and add background of clicking ripple animation.

**Comment by sharvaniharan** [comment]
Thats a good catch @cooltey will add the ripple and arrange it differently

**Comment by sharvaniharan** [comment]
@cooltey its done.. although i had to center it in a different way because Robin wants the click area to be large..

**Comment by cooltey** [comment]
> @cooltey its done.. although i had to center it in a different way because Robin wants the click area to be large..

I think add padding to the textView should be enough to enlarge the click area, isn't it?

**Comment by sharvaniharan** [comment]
the height of that parent layout itself is 48dp... so we can either just expand it to match parent and align center_vertical, what i have done... or have it wrap_content and give it a padding, and have the parent assign gravity... Is there any additional value to the second way...if there is, i will change it...

**Comment by cooltey** [comment]
> 
> 
> the height of that parent layout itself is 48dp... so we can either just expand it to match parent and align center_vertical, what i have done... or have it wrap_content and give it a padding, and have the parent assign gravity... Is there any additional value to the second way...if there is, i will change it...

It makes sense. But If you look closely, you'll find that the ripple animation of the reference text actually touches to the edge of the text, which looks weird to me and that's why I would recommend adding padding to the text.

**Comment by sharvaniharan** [comment]
There! @cooltey 😁

**Pull request merged**